### ISSUE TITLE
Helthcheck "Übersicht Kantonsebene"

### ISSUE THREAD

**Issue by akom-tipi** [digio-ch/pbs-healthcheck-core#49]
### Beschreibung
Der Healthcheck soll um eine Ansicht Übersicht Kantonsebene erweitert werden. Diese soll ähnlich wie die bestehende Ansicht auf Abteilungsebene aufgebaut sein und Einblicke in die Personen gewähren, die auf KV-Ebene aktiv sind.

### Akzeptanzkriterien
1. Neue Ansicht Übersicht Kantonsebene für Mitarbeitende auf KV-Ebene mit verschiedenen Widgets (Beschreibung der einzelnen Widgets unter Details).
2. Es kann aus allen Gremien im Kantonalverband selbst gewählt werden, welche Gremien in den Widgets angezeigt werden sollen.

### Details
1. Grafik mit Slots für die verschiedenen Gremien: Ein "leerer" Slot, für welchen aus einem Dropdown mit für den entsprechenden KV in der MiData erfassten Gremien ausgewählt werden kann, welches Gremium darin abgebildet werden soll. Für gewählte Gremium werden anschliessend alle in der MiData darin enthaltenen Personen aufgelistet mit Pfadiname, Alter, Geschlecht und Ausbildung (analog zur Ansicht für die Abteilungen). Es gibt keine vordefinierte Anzahl Slots, sobald für einen Slot ein Gremium ausgewählt wurde, soll ein neuer "leerer" Slot ergänzt werden. In der Liste zum Auswählen der Gremien soll wenn möglich die Hierarchie der Gremien aus der MiData abgebildet werden plus ein zusätzliches Gremium für die «Rollen auf KV Ebene» ergänzt werden.
2.  Kuchendiagramm Anzahl nach Geschlecht über alle ausgewählten Gremien 
3.  Kuchendiagramm Anzahl nach Abteilung über alle ausgewählten Gremien
5. GANTT-Diagramm Ebene KV, siehe #45 

### Offene Fragen
- Können Regionen gleich wie Kantonalverbände behandelt werden?

### Mockups
![mockup hc übersicht kv](https://user-images.githubusercontent.com/112755727/188266880-a32033c2-e474-4a7f-acb5-e16ac720b834.png)

**Comment by vouillamoz** [comment]
HC-195

**Comment by vouillamoz** [comment]
@taruxtin Wir haben mal ein Paar Fragen zusammengetragen. Evtl. erübrigen sich diese mit eurem nächsten Input. Auch könnten unsere Vorschläge/Gedanken eine neue Sicht reinbringen:

* Würde es Sinn machen die Filter Funktion auf eine Globale filter Funktion wie im HealthCheck zu ändern? (Statt lokalem filter bei Gruppen und Gremien Übersicht)
	* Ware das Dropdown immer das letze angezeigte "Gremium"? (UX technisch ungünstig bei scrolling mit vielen Gremien)
* Spezifikation Gruppen und Gremien Übersicht:
	* Welche Akzentfarbe haben die Gremien?
	* Kann man Gremien nur über das Dropdown wieder entfernen?
	* Ist eine optische Hervorhebung der Leiter/innen/ durch fette Schrift gut?
* Welche Rollen werden beim Gantt Diagramm angezeigt?
* Welche sind die Standard Rollen für das Gantt-Diagramm?
* Welche Gruppen sind die Hauptgruppen in Punkt 5, und welche sind ihre Farben?

**Comment by taruxtin** [comment]
>     * Würde es Sinn machen die Filter Funktion auf eine Globale filter Funktion wie im HealthCheck zu ändern? (Statt lokalem filter bei Gruppen und Gremien Übersicht)
>       * Ware das Dropdown immer das letze angezeigte "Gremium"? (UX technisch ungünstig bei scrolling mit vielen Gremien)
Ja, die Auswahl kann auch "global", d.h. über der Visualisierung der Gremien zugänglich gemacht werden. Dann erübrigt sich auch die zweite Frage.

>     * Spezifikation Gruppen und Gremien Übersicht:
>       * Welche Akzentfarbe haben die Gremien?
Könnte hier auf die Farben aus #53 zurückgegriffen werden.

>       * Kann man Gremien nur über das Dropdown wieder entfernen?
Ja.

>       * Ist eine optische Hervorhebung der Leiter/innen/ durch fette Schrift gut?
Wann müssen diese hervorgehoben werden?

>     * Welche Rollen werden beim Gantt Diagramm angezeigt?
Siehe #45 

>     * Welche sind die Standard Rollen für das Gantt-Diagramm?
Siehe #45 

>     * Welche Gruppen sind die Hauptgruppen in Punkt 5, und welche sind ihre Farben?
Gremien der obersten Ebene. Wie müssen noch klären, was wir machen, wenn Personen in mehreren Gremien drin sind.

Was in den AK noch fehlt: Die Auswahl der Gremien muss, vergleichbar wie die Rollen in #45 persistiert werden.
